src/main/java/carRacing/Main.java

```diff
@@ -0,0 +1,16 @@
+package carRacing;
+
+public class Main {
+
+    public static void main(String[] args) {
+        int carCount = InputView.initCarCount();
+        RacingGame racingGame = new RacingGame();
+        racingGame.initRacingCars(carCount);
```

Comment:
```suggestion
        RacingGame racingGame = new RacingGame(carCount);
```
위와 같이 구현할 경우 RacingGame을 사용하는 개발자가 init 메서드를 호출하지 않으면 자동차가 초기화가 되지 않을 수도 있지 않을까요?
반드시 필요한 초기화라면 생성자를 통해 초기화는 것은 어떨까요?

Reply:
반드시 초기화를 해야하는데, 메소드명처럼 의도를 표현하고 싶은 경우에는 어떻게 접근해보는게 좋을까요?
정적 팩토리 메소드가 그런경우에 필요한 부분일까요?